fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -2849,6 +2849,10 @@ public void setPlanNereidsDump(boolean planNereidsDump) {
         this.planNereidsDump = planNereidsDump;
     }
 
+    public void setDryRunQuery(boolean dryRunQuery) {
```

Comment:
why add this method?

Reply:
Because create prepared statement needs to modify the dryRun = true of ConnectContext SessionVeriable, and then execute the query to get the result schema, set dryRun = false when actually executing query.

Reply:
done